code/bsg.h

```diff
@@ -27,6 +28,7 @@ typedef struct BSGStruct {
   uint16_t rpm;           ///< BSG rotation (RPM)
   uint16_t voltage;       ///< BSG voltage (V)
   uint16_t current;       ///< BSG current (A)
+  uint16_t bsgReport;    ///< Time to report BSG status [S]
```

Comment:
Suggestion:  rename and replace with `time_t` from `time.h`. This is usually the standard type used to store time in C.
```suggestion
  time_t time;    ///< [s] Timestamp from last BSG status report
```

ref: https://en.cppreference.com/w/c/chrono/time_t